.github/workflows/unit-test.yml

```
@@ -2,34 +1,35 @@
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup GitHub Action
+      uses: actions/setup-node@v1
+      with:
+        node-version: 22.2.0
+    - name: Install dependencies
+      run: npm install
+    - name: Test
+      run: npm run unit_test:ci
+      env:
+        FYLE_CLIENT_ID: "lolo" 
```

yamllint

[error] trailing spaces

(trailing-spaces)